service/status-report/main.go

```diff
@@ -141,6 +151,16 @@ func main() {
 	if err != nil {
 		klog.Exitf("Failed to update mon configmap for storageClient %v: %v", storageClient.Status.ConsumerID, err)
 	}
+	joinSecret := &corev1.Secret{}
+	joinSecret.Name = "noobaa-remote-join-secret"
+	joinSecret.Namespace = operatorNamespace
+	joinSecret.Data = map[string][]byte{
+		"auth_token": []byte(noobaaAuthToken),
+		"mgmt_addr":  []byte(noobaaMgmtAddress),
+	}
+	if err = cl.Create(ctx, joinSecret); err != nil {
+		klog.Exitf("Failed to create join secret %s in namespace %s: %v", joinSecret.Name, joinSecret.Namespace, err)
+	}
```

Comment:
- We can create it where we find the secret like we are doing in other cases 
- Create will fail on the second reconcile. Should we use CreateOrUpdate instead?
- We should probably set owner ref on this as well

Reply:
yes true either use CreateOrUpdate or handle already exist case.

Reply:
removed secret creation in status-reporter